tests/test_user.py

```diff
@@ -218,6 +217,48 @@ def test_noop(self, requests_mock, cli_runner):
         cli_runner.invoke(change_command, ["1"])
         assert not requests_mock.called
 
+    @pytest.mark.usefixtures("mock_change_password_url")
+    def test_success_change_password(self, requests_mock, cli_runner):
+        new_password = "new_password"
+        data = {
+            "new_password1": new_password,
+            "new_password2": new_password,
+            "old_password": "old_password",
+        }
+        user_id = "1"
+
+        requests_mock.get(f"{USERS_URL}/{user_id}", json={"id": user_id})
+        requests_mock.post(
+            f"{CHANGE_PASSWORD_URL}", additional_matcher=partial(match_uploaded_json, data)
```

Comment:
This `additional_matcher` is not passing because on one side you are sending "old_password": "old_password" and on the other "old_password": "secret".